### DISCUSSION TITLE
php-qrcode : Unabled to create custom qrcode

### DISCUSSION THREAD

**Discussion by digitalbambou** [chillerlan/php-qrcode#176] [Q&A]
Hi,
I installed the latests release of the library via composer on my local webserver.
I'm using php8.1.
Basic use is working but i still have issues when i try to use custom output.

What is working (simple qrcode) :
`use chillerlan\QRCode\{QRCode, QROptions};
require_once .'/../vendor/autoload.php';
$options = new QROptions([
	'version'    => 5,
	'outputType' => QRCode::OUTPUT_MARKUP_SVG,
	'eccLevel'   => QRCode::ECC_L,
]);
$data = "https://myUrl.com";
$qrcode = new QRCode($options);
$renderQrcode = $qrcode->render($data);
echo '<img src="'.$renderQrcode.'" alt="QR Code" />';
exit();
`

What i want to do (custom qrcode whith logo, custom shape and custom colors):
I try to do it step by step.
First i'm using svgWithLogo.php example but i have a fatal error.
Fatal error: Uncaught Error: Class "chillerlan\QRCode\Output\QRMarkupSVG

I think there is a step that I am not doing correctly.
can i have any help ?

Regards and Happy new year !

**Comment by codemasher** [comment]
The examples you're trying to run are from `dev-main` and not included/incompatible with the `latest` release (v4.3.x). Try [installing the library](https://github.com/chillerlan/php-qrcode/wiki/Installation) from `dev-main` instead.

**Reply by digitalbambou** [reply]
yes i already read this post https://github.com/chillerlan/php-qrcode/discussions/135 but still have issues.
i'll try again

Thanks

**Reply by digitalbambou** [reply]
Thanks a lot Codemaster.
It works fine now. I can generate qrcode with or whitout svg logo, with or whitout custom colors and with square or rounded dots.
Now let's try to generate the same qrcode with png or jpg logos.
I'm going to try imageWithLogo Exemple

**Reply by codemasher** [reply]
I'm glad it works for you now! Thank you for the feedback!
